packages/components/heading/index.js

```diff
@@ -38,11 +39,11 @@ const H2 = props => (
   <Heading
```

Comment:
Heading needs till h6 right ?

Reply:
You are correct. But our current app doesn't have the styles for all of them.
If you want we can have the classification without styles.

Reply:
leave it then.

Reply:
We will add when required.